Monorepo (#88)
Shopify/web-foundation: packages/babel-preset/CHANGELOG.md

# Changelog

## [Unreleased]

> The babel preset is now scoped undo the `@shopify` name. This mean the name of the preset is now `@shopify/babel-preset`.

Reviewer: ```suggestion
The babel preset is now scoped under the `@shopify` name. This mean the name of the preset is now `@shopify/babel-preset`.
```

> The babel preset is now scoped undo the `@shopify` name. This mean the name of the preset is now `@shopify/babel-preset`.

Reviewer 2: We should make this more noticeable with a different header. Something like: 
## 🚨Package rename

# Changelog

## [Unreleased]

🚨Package rename

> The babel preset is now scoped under the `@shopify` name. This mean the name of the preset is now `@shopify/babel-preset`.

Reviewer 3: ```suggestion
This package has been renamed from `babel-preset-shopify` to `@shopify/babel-preset.
```
